src/instructlab/config.py

```diff
@@ -38,6 +38,17 @@
 # use spawn start method, fork is not thread-safe
 DEFAULT_MULTIPROCESSING_START_METHOD = "spawn"
 
+# When otherwise unknown, ilab uses this as the default family
+DEFAULT_MODEL_FAMILY = "merlinte"
+
+# Model families understood by ilab
+MODEL_FAMILIES = set(("merlinite", "mixtral"))
+
+# Map model names to their family
+MODEL_FAMILY_MAPPINGS = {
```

Comment:
I believe a tuple is more appropriate given the static nature of the mapping. How about:

```
MODEL_FAMILY_MAPPINGS = (
    ("granite", "merlinite"),
)
```

Reply:
That seems more complicated.  Instead of a direct lookup on the key, we'd have to iterate over all items checking for a match.

Reply:
I'm ok with the way it is.

Reply:
How much over head would be required to and more model families going forward? We will have others right?

Reply:
Seems like we would just need to flesh out this check, and ensure every supported model family has a proper mapping

Reply:
i expand some of my thoughts here: https://github.com/instructlab/instructlab/issues/1300